.travis.yml

```diff
@@ -5,12 +5,17 @@ jobs:
   include:
     - stage: analyze
       name: "SDK: dev; PKG: example; TASKS: `dartanalyzer .`"
-      dart: dev
```

Comment:
Don't remove `dev` version of Dart, flutter always uses `dev` version of dart. Flutter users might face issues/errors if we don't test in `dev` version